skt/reporter.py

```diff
@@ -60,12 +60,16 @@ def load_state_cfg(statefile):
     Args:
         statefile:      Path to a skt state file to read
 
+    Raises:
+        Exception in case statefile can't be read.
```

Comment:
Thank you, Veronika! Now we only have documentation to update :)

Reply:
Wait, what documentation do you mean? AFAIK we aren't listing which function can throw what exceptions anywhere

Reply:
Alright, this docstring can be read as "an exception", or `Exception`. If it's the former, then it's better to remove it, since it is of no use to the caller, because they can't catch the specific exception. If it's the latter, then it's just wrong, because we're not throwing `Exception` :) So, either specify the exception type and conditions it's being thrown at, or just remove the section :)

Reply:
It's supposed to be the second one. There are multiple subclasses of `Exception` that can be raised (eg. `OSError` if the file doesn't exist, `IOError` if we don't have the permission to read the file). I *think* it should be safe to name those two, but am not 100% sure if there is a possibility for a different one, so I went with the base class. I can remove it if you don't like the generic info though.

Reply:
Removed.